test/mail/encoders/quoted_printable_test.exs

```diff
@@ -0,0 +1,185 @@
+defmodule Mail.Encoders.QuotedPrintableTest do
+  use ExUnit.Case
+  import Mail.Encoders.QuotedPrintable
```

Comment:
I try not to rely on `import` in the libraries. I prefer a more verbose approach to remain explicit. I think it will make it easier for others looking to contribute.